packages/components/src/IllustratedMessage/src/IllustratedMessage.module.css

```diff
@@ -0,0 +1,38 @@
+.hop-IllustratedMessage {
+    display: grid;
```

Comment:
we try to create a third level of tokens, which is the "component tokens" inside the components. For everything that is relevant.

For instance, in this example
```
    --hop-IllustratedMessage-sm-size: 20rem;
    --hop-IllustratedMessage-md-size: 25rem;
    --hop-IllustratedMessage-lg-size: 28.5rem;


.hop-IllustratedMessage--sm {
    --max-width: var(--hop-IllustratedMessage-sm-size);
}
```

At some point, we could extract --hop-IllustratedMessage-sm-size in the tokens packages, so people that are using JS framework other than react could use those tokens to re-create our components.

--max-width is considered an "internal variable" and is used when the value is reassigned a couple of time. Those are not going to be exported